packages/atomic/package.json

```diff
@@ -20,8 +20,8 @@
     "build:locales": "node --max-old-space-size=4096 ./scripts/split-locales.js",
     "start": "npm run build:locales && stencil build --dev --watch --serve",
     "start:prod": "npx serve www -l 3333",
-    "test": "stencil test --spec --e2e",
-    "test:watch": "stencil test --spec --e2e --watchAll",
+    "test": "stencil test --spec",
```

Comment:
there are no e2e tests for atomic and we don't plan to add any, we will always use Cypress tests instead